app/templates/main/pagination.html

```diff
@@ -5,7 +5,7 @@
         {% if results.has_prev == True %}
             <div class="govuk-pagination__prev">
                 <a class="govuk-link govuk-pagination__link"
-                   href="{{ url_for(view_name, page=current_page-1, **filters) }}"
+                   href="{% if id != None %} {{ url_for(view_name, _id=id, page=current_page-1, **dict_args) }} {% else %} {{ url_for(view_name, page=current_page-1, **dict_args) }} {% endif %}"
```

Comment:
could we rename dict_args something a bit more meaningful as its quite vague at the moment?

Reply:
please suggest the name and I will replace it

Reply:
renamed dict_args to query_string_parameters at all places where dict_args used

Reply:
query_string_parameters is much clearer, thank you